src/main/java/com/synopsys/integration/detect/workflow/blackduck/developer/aggregate/RapidScanComponentGroupDetail.java

```diff
@@ -211,6 +207,50 @@ public void addVulnerabilityMessages(DeveloperScansScanView resultView,
         addMessages(errorMessage, warningMessage);
     }
 
+    // TODO rapid scans are now using the v5 developer-scans endpoint which no longer constructs
+    // warning and error messages. Until BlackDuck adds that back we have to construct our own messages.
+    // While it may be possible to reduce the overall message generation code in this class by pushing
+    // some common pieces into a parent class or interface, it is likely not worth altering the libraries
+    // as this may be temporary code.
```

Comment:
Yeah this might be safe to remove as I agree it is unlikely to change. At one point hub returned more data and we could simply pluck the pre-constructed error/warning messages directly from the JSON response. Now we have to build it.